src/index.js

```diff
@@ -82,14 +82,24 @@ options.vnode = vnode => {
 			}
 		}
 		else if (attrs) {
+			if (typeof vnode.nodeName==='string' && vnode.attributes && vnode.attributes.defaultValue) {
+				addDefaultValue(vnode);
```

Comment:
@gholtslander-va I know we started out with this as a separate function, but I'm thinking since it ended up being pretty small (and not creating a closure) we should inline it for bundle size.

```js
if (typeof vnode.nodeName==='string' && attrs && attrs.defaultValue) {
	if (!attrs.value && attrs.value!==0) {
		attrs.value = attrs.defaultValue;
	}
	delete attrs.defaultValue;
}
```

Seem reasonable?